src/Microsoft.AspNet.Server.Kestrel/Http/FrameRequestStream.cs

```diff
@@ -94,7 +94,7 @@ public Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationTok
                 {
                     tcs2.SetResult(task2.Result);
                 }
-            }, tcs);
+            }, tcs, cancellationToken);
             return tcs.Task;
```

Comment:
Ah yeah, make it private then; also it passes in `CancellationToken.None`, no harm in passing it along though.